operator/Makefile

```diff
@@ -271,12 +284,10 @@ delete-node:
 DELETE_CLUSTER ?= false
 .PHONY: clean
 clean: helm-uninstall uninstall-crds ## Clean up all resources.
```

Comment:
I've found that running `make clean` or `gradle clean` when I have no kind cluster completely fails. We'll need to address this, as it should definitely be possible to iterate on `gradle build / gradle clean` as a developer without ever running kind.